django/cantusdb_project/main_app/views/views.py

```diff
@@ -746,28 +747,53 @@ def json_node_export(request, id: int) -> JsonResponse:
 
 def notation_json_export(request, id: int) -> JsonResponse:
     """
-    Return all fields of the notation with the specified ID
+    Return JsonResponse containing several key:value pairs
+    for the notation with the specified ID
     """
 
-    notation_qs: QuerySet[Notation] = Notation.objects.filter(id=id)
-    if len(notation_qs) == 0:
-        return HttpResponseNotFound()
+    notation: Notation = get_object_or_404(Notation, id=id)
+
+    User = get_user_model()
+    created_by: Optional[User] = notation.created_by
+    last_updated_by: Optional[User] = notation.last_updated_by
+
+    data = {
+        "id": notation.id,
+        "name": notation.name,
+        "date_created": notation.date_created,
+        "date_updated": notation.date_updated,
+        "created_by": created_by.id if created_by else None,
+        "last_updated_by": last_updated_by.id if last_updated_by else None,
+    }
 
-    values: dict = dict(*notation_qs.values())
-    return JsonResponse(values)
+    return JsonResponse(data)
 
 
 def provenance_json_export(request, id: int) -> JsonResponse:
     """
-    Return all fields of the provenance with the specified ID
+    Return JsonResponse containing several key:value pairs
+    for the provenance with the specified ID
     """
 
-    provenance_qs: QuerySet[Provenance] = Provenance.objects.filter(id=id)
-    if len(provenance_qs) == 0:
+    try:
```

Comment:
this one doesn't use `get_object_or_404`?

Reply:
hm... something has gone wrong here. Let me investigate.

Reply:
I was so excited to update the docstring for the method, it seems, that I overlooked the raison d'être of the PR. Thanks for catching this.